src/app/Browser.js

```diff
@@ -95,30 +100,54 @@ function FileItem(props) {
 
   const copyButton = (
     <IconButton
+      title="Kopiera länk"
       style={{ boxShadow: 'none' }}
       hoveredStyle={{ boxShadow: 'none' }}
       onClick={(e) => {
         e.preventDefault()
         e.stopPropagation()
         copyToClipboard(name)
       }}>
-      <FontIcon
-        className="fas fa-link fa-xs"
-      >
-      </FontIcon>
+      <LinkIcon hoverColor="#ff5722" />
     </IconButton>
   )
 
+  
+  const isPublic = tags
+                  .filter(t => t["Key"] === "public")
+                  .map(t => t["Value"])[0]
+                  .toLowerCase() === "true"
```

Comment:
What is the purpose of the map here if we only access the first item in the list?

Reply:
Ahh, `find` seems useful